tests/Feature/Documents/DocumentsTest.php

```
@@ -0,0 +1,938 @@
+    private User $userWithoutPermission;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->bindFakeStorageProvider();
+
+        Integration::create([
+            'name'     => 'local',
+            'api_type' => 'file',
+        ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Models\Integration' in method 'setUp'. (undefined)

(StaticAccess)